pyfar/coordinates.py

```diff
@@ -1634,7 +1634,7 @@ def _make_kdtree(self):
     def __getitem__(self, index):
         """Return mutable slice of Coordinates object at index."""
```

Comment:
```suggestion
        """Return copied slice of Coordinates object at index."""
```

Mutable might suggest that the original changes, if you change the slice. This is not the case and it should not be the case. If it was the following could happen:
- create Coordinates object `c` in Cartesian coordinates
- make a slice of c and change it to spherical coordinates
- if the slice was mutable `c` would be a mixture of Cartesian and spherical coordinates...